trk/src/Screens/ClimbDetail/index.js

```diff
@@ -83,11 +126,11 @@ function ClimbDetail(props) {
       .catch((error) => {
         console.error("Error getting setter image URL: ", error);
       });
-  }, []);
+    }
+  }, [climbData]);
 
```

Comment:
this if statement deals with if we are navigating from home and have not yet fetched climbData (without this if statement, error of climbData.image is undefined)